src/Components/Login.js

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import Box from '@mui/material/Box'
+import TextField from '@mui/material/TextField'
+import Button from '@mui/material/Button'
+import Card from './Card.js'
+import './Login.scss'
+
+const Login = () => {
+
+    const [isUsernameError, setIsUsernameError] = useState(false)
+    const [isPassword1Error, setIsPassword1Error] = useState(false);
+    const [isPassword2Error, setIsPassword2Error] = useState(false);
+    const [isEmailError, setIsEmailError] = useState(false);
+    const [enteredPassword1, setEnteredPassword1] = useState('');
+    const [enteredPassword2, setEnteredPassword2] = useState('');
+    const [enteredEmail, setEnteredEmail] = useState('');
+    const [enteredUsername, setEnteredUsername] = useState('');
+
+    const usernameChangehandler = (event) => {
+        setEnteredUsername(event.target.value);
+    }
+
+    const password1ChangeHandler = (event) => {
+        setEnteredPassword1(event.target.value)
+    }
+
+    const password2ChangeHandler = (event) => {
+        setEnteredPassword2(event.target.value)
+    }
+    
+    const emailChangeHandler = (event) => {
+        setEnteredEmail(event.target.value)
+    }
+    
+    const validateEmailHandler = () => {
```

Comment:
This is an `onBlur` handler. Although this is purely preferential but for smaller forms we can do the validations on submit. That way we don't have to write so many functions